src/ontology/mondo-edit.obo

```diff
@@ -151913,25 +151913,51 @@ is_a: MONDO:0003847 ! inherited genetic disease
-name: porencephaly 1
+name: brain small vessel disease 1 with or without ocular anomalies
 def: "Any porencephaly in which the cause of the disease is a mutation in the COL4A1 gene." [MONDO:patterns/disease_series_by_gene]
+subset: ordo_disease {source="Orphanet:36383"}
+synonym: "ADT1P" EXACT [OMIM:175780]
+synonym: "autosomal dominant retinal arteriolar tortuosity, infantile hemiparesis, and leukencephalopathy" EXACT [] {comment="DOID:0090125"}
+synonym: "brain small vessel disease with Axenfeld-Riegar anomaly" EXACT [DOID:0090125]
+synonym: "brain small vessel disease with axenfeld-rieger anomaly" EXACT [OMIM:175780, OMIM:607595]
+synonym: "brain small vessel disease with hemorrhage" EXACT [DOID:0090125, OMIM:175780]
+synonym: "brain small vessel disease with or without ocular anomalies" EXACT [DOID:0090125, MONDO:Lexical, OMIM:607595]
+synonym: "brain small vessel disease with or without ocular anomalies; BSVD" EXACT [OMIM:607595]
+synonym: "BSVD" EXACT [DOID:0090125, MONDO:Lexical, OMIM:607595]
+synonym: "BSVD1" EXACT [OMIM:175780]
 synonym: "COL4A1 porencephaly" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "COL4A1-related brain small vessel disease with hemorrhage" EXACT [DOID:0090125, Orphanet:36383]
+synonym: "COL4A1-related familial vascular leukoencephalopathy" EXACT [OMIM:607595]
+synonym: "COL4A1-related retinal arteriolar tortuosity-infantile hemiparesis-autosomal dominant leukoencephalopathy syndrome" EXACT [DOID:0090125, Orphanet:36383]
+synonym: "hemiplegia, infantile, with porencephaly" EXACT [OMIM:175780]
 synonym: "hemiplegia, infantile, with porencephaly porencephaly, type 1" RELATED [OMIM:175780]
+synonym: "infantile hemiparesis" EXACT [DOID:0090125, OMIM:607595]
+synonym: "leukoencephalopathy with axenfeld-rieger anomaly" EXACT [DOID:0090125, OMIM:175780, OMIM:607595]
 synonym: "POREN1" RELATED [MONDO:Lexical, OMIM:175780]
-synonym: "porencephaly 1" EXACT [MONDO:Lexical, OMIM:175780]
+synonym: "porencephaly 1" EXACT [https://github.com/monarch-initiative/mondo/issues/1089, MONDO:Lexical, OMIM:175780]
 synonym: "porencephaly 1; POREN1" RELATED [OMIM:175780]
 synonym: "porencephaly caused by mutation in COL4A1" EXACT [MONDO:design_pattern]
 synonym: "porencephaly type 1" EXACT [MONDORULE:1, OMIM:175780]
 synonym: "porencephaly, type 1, autosomal dominant" RELATED [OMIM:175780]
+synonym: "retinal arteriolar tortuosity, infantile hemiparesis, and leukoencephalopathy, autosomal dominant" EXACT [OMIM:175780, OMIM:607595]
+synonym: "T1P" EXACT [OMIM:175780]
+xref: DOID:0090125 {source="MONDO:equivalentTo"}
+xref: ICD10:I67.3 {source="ORDO:36383/attributed", source="ORDO:36383/ntbt"}
+xref: MESH:C531642 {source="ORDO:36383/e", source="MONDO:equivalentTo"}
+xref: MESH:C564372 {source="MEDIC", source="MONDO:equivalentTo"}
 xref: OMIM:175780 {source="MONDO:equivalentTo"}
+xref: OMIM:607595 {source="ORDO:36383/e", source="MONDO:equivalentTo", source="Orphanet:36383", source="DOID:0090125"}
```

Comment:
note you are proxy merging two OMIMs. Consider assigning one obsoleteEquivalent

Reply:
also fix conflict

Reply:
done